BullsAndCows/SecretGenerator.cs

```diff
@@ -1,12 +1,27 @@
 ﻿using System;
+using System.Collections.Generic;
 
 namespace BullsAndCows
 {
     public class SecretGenerator
     {
         public virtual string GenerateSecret()
         {
-            throw new NotImplementedException();
+            var rand = new Random();
+            var numbers = new List<int>();
+
+            for (var i = 0; i < 6; i++)
```

Comment:
Why generate 5 numbers here?

Reply:
you are right, this cycle generate 6 numbers actually

Reply:
right